diff -ruN orig/dropbox.in new/dropbox.in
--- orig/dropbox.in	2020-04-14 17:19:34.000000000 -0600
+++ new/dropbox.in	2020-09-23 23:01:25.894015105 -0600
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 #
 # Copyright (c) Dropbox, Inc.
 #
diff -ruN orig/rst2man.py new/rst2man.py
--- orig/rst2man.py	2020-04-14 17:19:34.000000000 -0600
+++ new/rst2man.py	2020-09-23 23:03:43.042554870 -0600
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/python3
 
 # Author: 
 # Contact: grubert@users.sf.net
